.travis.yml

```diff
@@ -39,6 +40,7 @@ matrix:
 
   allow_failures:
     - php: hhvm
+    - php: 7.1
```

Comment:
Since the tests are passing on 7.1 do we need to keep it under `allow_failures`?